src/assets/carret-down.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can compress this SVG. [Here](https://jakearchibald.github.io/svgomg/), for instance.
Same applies to the other SVGs in this PR

Reply:
Hey this is a nice tool! I think it slimmed down the payloads by about 50%. I'm going to look for a way to have this integrated into the build step, done automatically. Thank you!